db/schema.rb

```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```

Comment:
I don't think we want to have this in here :-)

Reply:
i think we should consider keeping it! let's talk about this on Wednesday